assets/css/style.css

```diff
@@ -117,6 +117,18 @@ h1, h2, h3, h4, h5, h6 {
     display: none;
 }
 
+section .alert-success {
+    margin-top: 15px;
+    position: absolute;
+    background-color: #ccf6e4;
+    border-color: #b8f2da;
+    color: #006d3f;
+    width: 100%;
+    text-align: center;
+    z-index: 99;
+    display: none;
+}
+
```

Comment:
merge `master` branch in `feature/barbers` and moderate what happens here. 

( `feature/appointment` is now in `master` branch ) - [see changes you did](https://github.com/MirabbosMirzakhmedov/mroomapp/commit/13593686dbefaf76b0133d0db2716bfd9fb45139) in `feature/appointment` for this CSS

Reply:
Done

Reply:
Resolved.

Reply:
Done